map.js

```diff
@@ -416,6 +519,29 @@ function worldSpaceToMapSpace(point){
     return new Point(map_x, map_y);
 }
 
+//really only useful for debugging
+/**
+ * 
+ * @param {Point} point 
+ */
+function mapSpaceToWorldSpace(point){
```

Comment:
I could actually see this being useful if someone ever wants to map out locations on the up scaled image instead of using over world cords. 

But for now, yeah, i don't see any use beyond debugging. But still nice to have here. Nice job.